backend/validators/llm.py

```
@@ -62,11 +62,28 @@ def __del__(self):
 
     async def stop(self):
         if self._process is not None:
+            print(f"[LLMModule] Stopping process (PID: {self._process.pid})")
             try:
                 self._process.send_signal(signal.SIGINT)
             except ProcessLookupError:
                 pass
```

Ruff (0.12.2)

Use `contextlib.suppress(ProcessLookupError)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(ProcessLookupError)`

(SIM105)